compiler/control/OMROptions.cpp

```diff
@@ -100,6 +100,7 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
         TR::Options::setStaticNumeric, (intptrj_t)&OMR::Options::_abstractTimeToReduceInliningAggressiveness, 0, "F%d", NOT_IN_SUBSET },
    {"acceptHugeMethods",     "O\tallow processing of really large methods", SET_OPTION_BIT(TR_ProcessHugeMethods), "F" },
    {"activateCompThreadWhenHighPriReqIsBlocked", "M\tactivate another compilation thread when high priority request is blocked",  SET_OPTION_BIT(TR_ActivateCompThreadWhenHighPriReqIsBlocked), "F", NOT_IN_SUBSET},
+   {"aggressiveLoopVersionerOpts", "O\tUse more loop versioning", SET_OPTION_BIT(TR_aggressiveLoopVersionerOpts), "F" },
```

Comment:
This option sounds weird because its not in an imperative mood mood. IMO options should be commands. Would `aggressivelyVersonLoops` not sounds more imperative?